pages/about/about.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<section id="about">
+  <md-card layout-fill class="oppia-foundation-vertical-padding oppia-foundation-green-background">
+    <md-card-content layout="column" layout-align="center center" class="oppia-foundation-white-text">
+      <md-content layout-wrap class="oppia-foundation-green-background oppia-foundation-white-text">
+        <center>
+          <img ng-src="/static/images/ _85px_white_mintbkgd.png" style="width: 250px;">
```

Comment:
Is there supposed to be a space before the _ ?

Consider renaming the image to something like about_page_logo.png or similar.